path_handler.py

```diff
@@ -0,0 +1,15 @@
+import os
```

Comment:
Update: Okay I see that is not just used for testing, but this shouldn't be used for the source. It technically brings a coupling of test code and source code. Also, the pathing issue isn't a problem for the framework.